app/views/works/_work_module.html.erb

```diff
@@ -46,6 +46,9 @@
     <p class="datetime"><%= set_format_for_date(work.revised_at) %></p>
   </div>
 	  
+  <% if !work.posted? %>
+   <p class="caution notice"><%= ts('This draft will be <strong> automatically deleted </strong> on %{time}',time:  time_in_zone(work.created_at + 1.month)).html_safe %></p>
```

Comment:
Ersatz-Hound says: Missing space after comma. Extra space between key and value.

Also extra space between HTML tag and what's inside it.